src/Microsoft.AspNetCore.Razor.Evolution/Properties/AssemblyInfo.cs

```diff
@@ -1,14 +1,7 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.Reflection;
-using System.Resources;
 using System.Runtime.CompilerServices;
 
-[assembly: AssemblyMetadata("Serviceable", "True")]
```

Comment:
These attributes are all generated by the SDK when MSBuild attributes are present. See https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.GenerateAssemblyInfo.targets